vectorstores/pgvector/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package pgvector contains an implementation of the vectorStore
```

Comment:
do you mean the `VectorStore` interface?

Reply:
change all `vectorStore` -> `VectorStore` in doc.go in commit 46b24e7e50be901081b57a8cdb87f640fe4c4c76